packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -824,10 +909,81 @@ export const prepareCosmosOrchestrationAccountKit = (
             ],
             restOpts,
           );
-          return this.facets.helper.overrideVow(
-            results,
-            harden({ status: 'unknown' }),
-          );
+
+          const { progressTracker } = restOpts;
+          if (progressTracker) {
+            const priorReport = progressTracker.getCurrentProgressReport();
+            /** @type {TrafficEntry} */
+            const transferTraffic = {
+              op: 'transfer',
+              src: [
+                'ibc',
+                ['chain', `cosmos:${chainAddress.chainId}`],
+                ['port', transferChannel.portId],
+                ['channel', transferChannel.channelId],
+              ],
+              dst: [
+                'ibc',
+                ['chain', `cosmos:${destination.chainId}`],
+                ['port', transferChannel.counterPartyPortId],
+                ['channel', transferChannel.counterPartyChannelId],
+              ],
+              seq: { status: 'pending' }, // filled in by fillSequenceWatcher.onFulfilled below
+              incomplete: true,
+            };
+            const report = {
+              ...priorReport,
+              traffic: [...(priorReport.traffic || []), transferTraffic],
+            };
+            const trafficEntryIndex = report.traffic.length - 1;
+            progressTracker.update(harden(report));
```

Comment:
Please bind late as possible so reader isn't looking for how it's used.

```suggestion
            progressTracker.update(harden(report));
            const trafficEntryIndex = report.traffic.length - 1;
```

Even better, just inline it.

Reply:
:+1: